uiowa/uiowa
# [Alerts] Ability to schedule publishing/unpublishing

Resolves https://github.com/uiowa/uiowa/issues/6637

# How to test

1. Schedule an alert to publish/unpublish.   
2. Run cron and verify that alert has been published or unpublished. 

## Test on default
```
ddev composer install && ddev blt ds --site=default && ddev drush @default.local config-split:activate sitenow_alerts -y && ddev drush @default.local uli node/add/alert
```
- Verify that editor, publisher, webmaster can all schedule alerts. 

## Test on facilities
```
ddev composer install && ddev blt ds --site=facilities.uiowa.edu && ddev drush @facilities.local uli node/add/alert
```
- Verify that editor, publisher, webmaster can all schedule alerts + collaborator.
- Verify that custom publish date field `field_alert_publish_on` has been removed from facilities site split.
